### DISCUSSION TITLE
Remove GitHub App as authentication device

### DISCUSSION THREAD

**Discussion by theking2** [community/community#12785] [Mobile] [answered]
For some time now the Android GitHub app stopped authenticating me. How do I remove GitHub App Authentication from my GitHub account?

**Comment by eliperkins** [comment, marked as answer]
You can revoke access for the [GitHub for iOS](https://github.com/settings/connections/applications/2cfa9b7a1b57de32dd0d) or [GitHub for Android](https://github.com/settings/connections/applications/3f8b8834a91f0caad392) OAuth apps from Settings > Applications > Authorized OAuth Apps and revoking the specific app from there.

> For some time now the Android GitHub app stopped authenticating me

I'd love to hear more about the issues you're running into here! What do you mean by "authenticating" in this context?

**Reply by robertIsaac** [reply]
> You can revoke access for the [GitHub for iOS](https://github.com/settings/connections/applications/2cfa9b7a1b57de32dd0d) or [GitHub for Android](https://github.com/settings/connections/applications/3f8b8834a91f0caad392) OAuth apps from Settings > Applications > Authorized OAuth Apps and revoking the specific app from there.
> 
> > For some time now the Android GitHub app stopped authenticating me
> 
> I'd love to hear more about the issues you're running into here! What do you mean by "authenticating" in this context?

how to remove only one of the two mobile devices I have?

**Reply by ghost** [reply]
* Login to Github in your browser
* User icon > Settings > Password and Authentication > Two-factor methods > Github Mobile > Show
* Scroll down to Github mobile sessions > Revoke on the smartphone to remove.
